joint_trajectory_controller/src/joint_trajectory_controller.cpp

```diff
@@ -607,6 +627,41 @@ void JointTrajectoryController::feedback_setup_callback(
     std::bind(&RealtimeGoalHandle::runNonRealtime, rt_active_goal_));
 }
 
+void JointTrajectoryController::sort_to_local_joint_order(
+  std::shared_ptr<trajectory_msgs::msg::JointTrajectory> trajectory_msg)
+{
+  // rearrange all points in the trajectory message based on mapping
+  std::vector<size_t> mapping_vector = mapping(trajectory_msg->joint_names, joint_names_);
+  auto remap = [](const std::vector<double> & to_remap, const std::vector<size_t> & mapping)
+    -> std::vector<double>
+    {
+      if (to_remap.size() != mapping.size()) {
+        return to_remap;
+      }
+      std::vector<double> output;
```

Comment:
```suggestion
      std::vector<double> output{};
```

Reply:
imo it looks cleaner without the empty initializer list

Reply:
I agree

Reply:
why would it be clearer?

Reply:
Why write more to denote *nothing*?

Reply:
This is clearly a nitpick issue we can follow up at a different time